public/stylesheets/style.css

```diff
@@ -0,0 +1,8 @@
+body {
```

Comment:
Should have specified a bit more that this is a API only and no client will be rendered. Sorry about that :) Can just remove the html-rendering logic / libraries